README.md

```diff
@@ -20,6 +20,8 @@ The attributes listed below are used in *components.json* to configure **Hotgrid
 
 **instruction** (string): This optional text appears above the component. It is frequently used to guide the learner’s interaction with the component.  
 
+**_canCycleThroughPagination** (boolean): Enables the pop-ups to be cycled through endlessly using either the previous or next icon. When set to `true`, clicking "next" on the final stage will display the very first stage. When set to `false`, the final stage will display only a "previous" icon. The default is `false`.  
```

Comment:
`_canCycleThroughPagination` needs adding to example.json and properties.schema as well

also need to add `_hidePagination` to all three